pom.xml

```diff
@@ -236,7 +236,7 @@
 				<jdk>[11,)</jdk>
 			</activation>
 			<properties>
-				<javadoc.opts>-Xdoclint:none -html5</javadoc.opts>
+				<javadoc.opts>-Xdoclint:none -html5 -tag implNote:a:"Implementation Note"</javadoc.opts>
```

Comment:
This is essentially adding implNote as a custom tag. It's a workaround fix but it really _shouldn't_ be necessary, as far as I know.

Reply:
Not that I object to a workaround fix if that gets our deployment unstuck, mind you :)